### ISSUE TITLE
[Bug]: body unable to scroll after closing alert dialog which is opened from the drawer

### ISSUE THREAD

**Issue by xeonicca** [radix-vue/shadcn-vue#507]
### Reproduction

https://github.com/xeonicca/drawer-dialog-demo

### Describe the bug

This bug happens when the alert dialog is opened when the drawer is already opened, after attempting to close the alert dialog with the `AlertDialogCancel` component and use a click handler to close the drawer at the same time, the body becomes unscrollable  due to `overflow: hidden` remained in inline style.

### System Info

```bash
System:
    OS: Windows 10 10.0.19045
    CPU: (6) x64 AMD Ryzen 5 3500X 6-Core Processor
    Memory: 21.71 GB / 47.95 GB
  Binaries:
    Node: 18.17.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - C:\Program Files (x86)\Yarn\bin\yarn.CMD    
    npm: 9.4.2 - C:\Program Files\nodejs\npm.CMD
    pnpm: 8.15.4 - ~\AppData\Roaming\npm\pnpm.CMD
  Browsers:
    Chrome: 123.0.6312.107
    Edge: Chromium (123.0.2420.81)
    Internet Explorer: 11.0.19041.3636
  npmPackages:
    radix-vue: ^1.7.1 => 1.7.1
    vue: ^3.4.21 => 3.4.23
```

### Contributes

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests

**Comment by sadeghbarati** [comment]
https://github.com/radix-vue/vaul-vue/issues/37

**Comment by zernonia** [comment]
Should be fixed with  `vaul-vue v0.1.2`
